addons/expanded-backpack/addon.json

```
@@ -1,5 +1,5 @@
 {
-  "name": "Multiple rows in backpack",
+  "name": "Multiple backpack rows",
```

Good, but I think this might be a little clearer:
```suggestion
  "name": "Multi-row backpack",
```